### PR TITLE
Add AV1 video format

### PR DESCRIPTION
**Pull request by valib**

### CODE REVIEW

**Comment by SubJunk** [comment]
@valib can we find a small sample file to add for testing? I think it would be good to add them whenever we add detection for a new format

File: src/main/java/net/pms/dlna/LibMediaInfoParser.java [hunk 1]
```diff
@@ -696,6 +696,8 @@ protected static void setFormat(StreamType streamType, DLNAMediaInfo media, DLNA
 				format = FormatConfiguration.VC1;
 		} else if (value.equals("au") || value.equals("uLaw/AU Audio File")) {
 			format = FormatConfiguration.AU;
+		} else if (value.equals("av01")) {
```
Comment (SubJunk) [inline src/main/java/net/pms/dlna/LibMediaInfoParser.java:699]: The string I can see in the MediaInfo output is `av1` not `av01`, is there a reason to have it like this?

Comment (valib) [reply src/main/java/net/pms/dlna/LibMediaInfoParser.java]: @SubJunk `AV1` is a Format and Codec ID is `av01`. I updated the parsing

**Comment by SubJunk** [comment]
Nice work!

**Comment by SubJunk** [comment]
@valib it would be good to add that sample you mentioned that gets identified differently too. If the file is too big I can try splitting it

**Comment by valib** [comment]
@SubJunk done in #1983

**Comment by SubJunk** [comment]
@valib thanks!

**Pull request merged**